src/slider.js

```diff
@@ -191,15 +185,16 @@ const Slider = ({
       <div css={styles.active} style={valueStyle} />
       <div
         ref={handle}
-        css={styles.thumb}
-        style={thumbStyle}
+        style={handleStyle}
         onTouchStart={handleMouseDown}
         onMouseDown={handleMouseDown}
         onClick={function(e) {
           e.stopPropagation();
           e.nativeEvent.stopImmediatePropagation();
         }}
-      />
+      >
+        <div css={styles.thumb} />
```

Comment:
With this approach we can provide a way to user pass the **thumb** as a custom component too.

Reply:
yep, probably add that later